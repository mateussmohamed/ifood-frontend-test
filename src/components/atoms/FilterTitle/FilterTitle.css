.filter-title {
  font-weight: 300;
}
