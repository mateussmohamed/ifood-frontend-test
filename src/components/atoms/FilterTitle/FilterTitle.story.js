import { storiesOf } from '@storybook/react';
import React from 'react';
import FilterTitle from './index';

const stories = storiesOf('FilterTitle', module);
stories.add('whit text prop', () => (
  <div style={{ float: 'left' }}>
    <FilterTitle text="Filtrar por" />
  </div>
));
