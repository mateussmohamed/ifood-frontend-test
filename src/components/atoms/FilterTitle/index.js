import React from 'react';
import PropTypes from 'prop-types';
import './FilterTitle.css';

const FilterTitle = ({ text }) => <div className="filter-title">{text}</div>;

FilterTitle.propTypes = {
  text: PropTypes.string.isRequired
};

export default FilterTitle;
