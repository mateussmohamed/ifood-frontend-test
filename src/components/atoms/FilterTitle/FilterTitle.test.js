import React from 'react';
import renderer from 'react-test-renderer';
import FilterTitle from './index';

describe('FilterTitle', () => {
  test('should FilterTitle match to snapshot', () => {
    const component = renderer.create(<FilterTitle text="awesome text" />);
    const tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

  test('should when text prop render the correct text', () => {
    const component = renderer.create(<FilterTitle text="awesome text" />);
    expect(component.toTree().props.text).toBe('awesome text');
  });
});
