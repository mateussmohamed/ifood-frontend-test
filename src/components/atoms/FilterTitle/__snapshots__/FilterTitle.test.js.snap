// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FilterTitle should FilterTitle match to snapshot 1`] = `
<div
  className="filter-title"
>
  awesome text
</div>
`;
